### PR TITLE
No need to do type cast for the same type

### PR DESCRIPTION
**Pull request by xiangpengzhao**
**What this PR does / why we need it**:
Just a nit. The two variables are the same type. No need to do type cast.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kube-proxy/app/server.go [hunk 1]
```diff
@@ -410,7 +410,7 @@ func createClients(config componentconfig.ClientConnectionConfiguration, masterO
 	kubeConfig.ContentType = config.ContentType
 	kubeConfig.QPS = config.QPS
 	//TODO make config struct use int instead of int32?
```
Comment (xiangpengzhao) [inline cmd/kube-proxy/app/server.go]: Is it worthy fixing this TODO? There might be tens of files to change.

Comment (BenTheElder) [reply cmd/kube-proxy/app/server.go]: this seems to be breaking CI batching https://github.com/kubernetes/test-infra/issues/5518

**Comment by BenTheElder** [comment]
/hold
I think this is causing excessive batch job failures which is really backing up the submit queue: https://github.com/kubernetes/test-infra/issues/5518

**Comment by xiangpengzhao** [comment]
/close
as per https://github.com/kubernetes/test-infra/issues/5518#issuecomment-344487762

**Pull request closed without merging**